### DISCUSSION TITLE
New stage features.

### DISCUSSION THREAD

**Discussion by oppotaku** [ikemen-engine/Ikemen-GO#892] [Feature Request] [answered]
Make it easier for a user to add platforms to stages, and allow the player to stand on one without changing the characters code.

Add the ability to be knocked off stage/ fall off stage/ hit away from 
stage.

Ring Outs, from real bout fatal fury.

**Comment by Windblade-GR01** [comment, marked as answer]
Platforms are already worked on and will be implemented in the future.
